src/nupic/algorithms/Anomaly.cpp

```diff
@@ -0,0 +1,114 @@
+/* ---------------------------------------------------------------------
+ * Numenta Platform for Intelligent Computing (NuPIC)
+ * Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+ * with Numenta, Inc., for a separate license for this software code, the
+ * following terms and conditions apply:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ *
+ * http://numenta.org/licenses/
+ * ---------------------------------------------------------------------
+ */
+
+
+#include <vector>
+#include <numeric>
+#include <algorithm>
+#include <set>
+
+#include "nupic/algorithms/Anomaly.hpp"
+#include "nupic/utils/Log.hpp"
+#include "nupic/utils/MovingAverage.hpp"
+
+using namespace std;
+
+namespace nupic
+{
+  namespace algorithms
+  {
+    namespace anomaly
+    {
+
+
+float computeRawAnomalyScore(const vector<UInt>& active, const vector<UInt>& predicted)
```

Comment:
The namespaces on lines 35-40 are indented for each new namespace but you aren't indenting the rest of the code. I am fine if you don't want to indent for namespaces be then be consistent in the lines above as well.